static/js/components/ChannelSidebar.js

```diff
@@ -30,7 +30,8 @@ const ChannelSidebar = ({ channel, isModerator }: ChannelSidebarProps) => {
         ) : null}
         <Markdown
           source={
-            channel.description || "(There is no description of this channel)"
+            channel.public_description ||
```

Comment:
I may be missing something, but I'm not quite sure I'm following this change here for a few reasons:

- The issue this is for doesn't describe removing the current description field, just adding the headline
- Markdown doesn't make any sense for a value that is limited to 80 characters

Reply:
I'll revert this, it's covered in a different PR I think, #989